web/app/components/comment/comment.jsx

```diff
@@ -413,7 +371,7 @@ export default class Comment extends Component {
             {
               isAdmin && mods.view !== 'user' && (
                 <span
-                  onClick={o.user.verified ? this.onUnverifyClick : this.onVerifyClick}
+                  onClick={() => this.toggleVerify(o.user.verified)}
```

Comment:
Click handler recreated on each render here. Though, I don't care. Since fixing this would be a premature optimization. Just wanted to note that bind in constructor is useless in this case.

Reply:
We also need to apply here a11y helper from #130

Reply:
Done https://github.com/umputun/remark/pull/126/commits/fe07a15498ed730ff7d9637630c236a3ee8ee8e0